test/step-function.test.ts

```diff
@@ -80,6 +80,77 @@ test("return PropAccessExpr", () => {
   expect(normalizeDefinition(definition)).toMatchSnapshot();
 });
 
+test("return ElementAccessExpr", () => {
```

Comment:
How sure are we that these new tests still run properly? Is there value in having snapshot tests without runtime tests?

Reply:
Generally I have been adding more verbose and granular tests here and adding similar, but less granular cases to the runtime tests.